src/web_interface/frontend_main.py

```diff
@@ -27,10 +27,11 @@ def __init__(self, config=None):
         logging.info("Web front end online")
 
     def _setup_app(self):
+        # TODO replace __name__ see Flask constructor docs
         self.app = Flask(__name__)
         self.app.config.from_object(__name__)
 
-        Flask.secret_key = os.urandom(24)
```

Comment:
I wonder how this worked, since it does not actually set the `secret_key` in `app.config`